drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/wizard/pages/ActionInsertFactFieldsPageViewImpl.ui.xml

```diff
@@ -26,90 +27,99 @@
   <ui:with field="res" type="org.uberfire.ext.widgets.core.client.resources.WizardResources"/>
 
   <gwt:VerticalPanel ui:field="container" width="100%">
```

Comment:
I would try to avoid using Vertical and Horizontal panels as possible. Using the Bootstrap grid system should be enough for composing  our widgets, with the plus of also being responsive. Have a look in this simple example here: https://github.com/cristianonicolai/kie-wb-common/blob/f5554d0037327c4d3806d1288958df2f284e4aff/kie-wb-common-widgets/kie-wb-common-ui/src/main/java/org/kie/workbench/common/widgets/client/popups/about/AboutPopup.ui.xml